### ISSUE TITLE
Continuous syncing

### ISSUE THREAD

**Issue by johan974** [k9mail/k-9#2544]
### Expected behavior
All auto sync options I have cleared. So on request new mail is retrieved

### Actual behavior
There is a continuous synchronisation for an account. 
I have the very same settings for the official K-9 Google play mail. 

### Steps to reproduce
1. I forked the k-9 sources, built them and installed them via Android Studio on my mobile Samsung S4. 
2. After exporting the settings of the K-9 Google Play mail, I imported them in the just build version. 
3. All mail is retrieved. There is a continuous synchronisation ... which will not be good for my mobile's battery. I did specify NOT to auto sync 

### Environment
K-9 Mail version: just retrieve and build sources. 

Android version: Cyanogenmod CM13. 

Account type (IMAP, POP3, WebDAV/Exchange): IMAP.

**Comment by cketti** [comment]
What do you expect us to do about this? You're not specifying which version you're using. You're using an unofficial Android build. And you didn't provide any information (like a debug log) that could hint at the source of the issue. Also, what does "continuous synchronisation" mean? What exactly are you looking at?

**Comment by johan974** [comment]
OK, I will try to be more specific. 
Version: I just retrieved the source version from github, built it via AS and deployed it to my mobile. So, this should be the newest version. 
The issue is: the synchronisation keeps active. So the 2 round errors of the synchronisation keep on spinning. The app keeps busy synchronizing. 
Expected behaviour: like the Google play version, when I manually pull down ... the synchronisation starts and stops after a few seconds. In this case the synchronisation continues. I see a spinning synchronisation circle in the top bar of my mobile. 
Did I set wrong settings? I don't think so, because I just imported the settings via the official Google Play version. 
Debug log: OK, I will provide in a minute.
